standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestHiveMetaStoreTimeout.java

```diff
@@ -117,11 +117,11 @@ public void testTimeout() throws Exception {
 
   @Test
   public void testResetTimeout() throws Exception {
-    HMSHandler.testTimeoutValue = 250;
     String dbName = "db";
 
     // no timeout before reset
     client.dropDatabase(dbName, true, true);
+    HMSHandler.testTimeoutValue = 250;
```

Comment:
I don't get it how this config will resolve the issue. Could you please elaborate it?

Reply:
Originally the low timeout value was set at the beginning of the test. Normally it is high enough that `client.dropDatabase(dbName, true, true);` won't fail, but `client.createDatabase(db);` will do. However, if the network/DB is very slow, even `client.dropDatabase(dbName, true, true);` can fail causing the whole test to fail. Since the default value of the timeout is -1 (no timout) I moved the lowering the value after the dropdatabase call, so now it is no longer sensitive to the environment.

Reply:
Good idea, I'll rewrite the test methods that way